priv/repo/migrations/20230606152133_add_ensa_version_to_sap_system_read_model.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Trento.Repo.Migrations.AddEnsaVersionToSapSystemReadModel do
+  use Ecto.Migration
+
+  def change do
+    alter table(:sap_systems) do
+      add :ensa_version, :string, default: "no_ensa"
```

Comment:
Setting already a default correct value to have the read models correctly populated

Reply:
I'm not a really fan of settings the defaults in the schema migration, this looks more like a hybrid within a data migration and a schema migration.

If we really need that, and this could no be enforced at application level, I will suggest another migration which will take care only of migrating data, setting the default value

Wdyt?

Reply:
How can I enforce this in the application?
I simply need a value that is within the new `EnsaVersion` enum for the scenario that it was not discovered.
Otherwise the database is initialized with `nil` until we get the event that changes this breaking the openapi schema, and in general the usage of the Enum

Reply:
You are talking about the already present sap systems when the migration will run?

Reply:
Oh okay, so you can create another migration, just with the data migration and leave the schema migration you already created without the default value.

That way you don't alter the schema with the default value (we could talk about having something default if we switch to postgres enums, but I don't recommend it, they are a pain to mantain and evolve), and leave a separate migration which migrates only data

Reply:
Done!
I have added the update in a new file to make it more obvious.
Ecto doesn't have any `update` command, so the `execute` with the basic SQL command looks the best solution